src/sentry/api/endpoints/project_plugin_reset.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import absolute_import
+
+from sentry.api.bases.project import ProjectEndpoint
+from rest_framework.response import Response
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.plugins import plugins
+from sentry.api.serializers import serialize
+from sentry.api.serializers.models.plugin import (
+    PluginWithConfigSerializer
+)
+
+
+class ProjectPluginResetEndpoint(ProjectEndpoint):
```

Comment:
rather than adding another endpoint for this, can you update [this](https://github.com/getsentry/sentry/blob/master/src/sentry/api/endpoints/project_plugin_details.py#L47) to just have a `reset` param?

Reply:
I feel like it's cleaner having a separate endpoint for it. It's kind of unexpected to me for a post request to just update an existing resource rather than create a new one.